### PR TITLE
Upgraded ingest-api version

### PR DESCRIPTION
**Pull request by emmberk**

### CODE REVIEW

**Comment by emmberk** [comment]
`./gradlew clean build` and `./gradlew dependencyCheckAnalyze --info` succeed for me locally. I'm overriding the CI build because jenkins is currently being restarted.

File: ingest/build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ plugins {
 dependencies {
 	testCompile group: "org.springframework.boot", name: "spring-boot-starter-test", version: "2.1.5.RELEASE"
 	compile group: "org.json", name: "json", version: "20180813"
-	implementation("com.connexta.ingest:ingest-api-rest-spring-stubs:0.1.0-SNAPSHOT")
+	implementation("com.connexta.ingest:ingest-api-rest-spring-stubs:0.1.1-SNAPSHOT")
```
Comment (Kjames5269) [inline ingest/build.gradle]: ❗️ I'm running the release for 0.1.1 right now, so we should wait until that passes and drop the `-SNAPSHOT`

**Comment by Kjames5269** [comment]
build now

**Comment by emmberk** [comment]
```
> Could not resolve all files for configuration ':ingest:compileClasspath'.
    > Could not find com.connexta.ingest:ingest-api-rest-spring-stubs:0.1.1.
      Searched in the following locations:
        - file:/root/.m2/repository/com/connexta/ingest/ingest-api-rest-spring-stubs/0.1.1/ingest-api-rest-spring-stubs-0.1.1.pom
        - file:/root/.m2/repository/com/connexta/ingest/ingest-api-rest-spring-stubs/0.1.1/ingest-api-rest-spring-stubs-0.1.1.jar
        - https://repo.maven.apache.org/maven2/com/connexta/ingest/ingest-api-rest-spring-stubs/0.1.1/ingest-api-rest-spring-stubs-0.1.1.pom
        - https://repo.maven.apache.org/maven2/com/connexta/ingest/ingest-api-rest-spring-stubs/0.1.1/ingest-api-rest-spring-stubs-0.1.1.jar
      Required by:
          project :ingest
```

**Comment by Kjames5269** [comment]
build now

**Comment by emmberk** [comment]
The CI build failed on failing to download CVEs, which is a known issue. I'm overriding the CI build because this has been tested locally.

**Pull request merged**